src/index.js

```diff
@@ -0,0 +1,26 @@
+import './style.css';
+import {
+  addTodo, displayList, getData, getEdit, saveEdit,
+} from './function.js';
```

Comment:
- [ ] [OPTIONAL] You could import all the functions from the `functions.js` file using `*` 
E.g.: `import * as Functions from './functions.js'`. In this case, if you need to call the `addTodo` function, you would use `Functions.addTodo()`